### ISSUE TITLE
Coil proximity missing from PDOM description if only 4 loop coil

### ISSUE THREAD

**Issue by jhung** [phetsims/faradays-law#131]
In the PDOM description of the Bar Magnet, if only the 4 loop coil is showing, the description of the magnet proximity to the coil is missing.

Currently:
"The bar magnet is at the bottom-center of the Play Area."

Should be:
"The bar magnet is at the bottom-center of the Play Area. [Proximity] 4 loop coil."

**Comment by terracoda** [comment]
@jhung, the proximity to coil is the most important information. I think that information should always come first if at all possible. Have you considered flipping the phrasing? For example:

Bar magnet. {{Proximity to coil}}, at {{relevant place in Play Area, only when needed}}.

<img width="601" alt="screen shot 2018-09-24 at 11 51 27 am" src="https://user-images.githubusercontent.com/1902116/45957707-36f6cb80-bff0-11e8-8b91-a5d5faf6c19c.png">

I would recommend:
- "Bar magnet. {{Just left of 4-loop coil}}, at center of Play Area."

In the case, above, you would get the string "Bar magnet" from the accessible name on focus, and then after that, maybe not hear it, but only hear your relevant proximity to the coil and possibly your relevant place in the Play Area - and the Play Area position only if needed, for example if you are far fro the coil. 

I think you already mentioned you were dropping certain instances of the string that describes the exact location in Play Area.

So to keep going in the case above, on the next few key press right, a user might hear:
- - "{{In 4-loop coil}}." and hear a big sonification representing the flux.
- - "{{In 4-loop coil}}." and hear a big sonification representing the flux.
- - "{{Just right of 4-loop coil}}." and hear the smaller flux sound, hopefully indicating that flux is stronger when you move and even stronger inside the actual coil.

Of course, my suggestions for "just right" and "just left" might not be the best phrases depending on how the positions are described, but I think it makes sense to describe the position of the magnet always in relation to its proximity to the coil or coils. And I think you might already be iterating in this direction.

**Comment by terracoda** [comment]
And apologies if this should be entirely new issue.

**Comment by terracoda** [comment]
I think this issue and issues #133 and #134 might need to be considered together as a group, so just tagging those issues here.

**Comment by jhung** [comment]
We should put this on hold pending discussions regarding an explicit magnet pick-up and release interaction.

**Comment by jhung** [comment]
No longer applicable with revised designs. Closing this issue.
